### ISSUE TITLE
P5 not going blank on 'Stop Now'

### ISSUE THREAD

**Issue by Cboehm00** [FalconChristmas/fpp#773]
When playing a sequence in 4.1 on my P5, the screen does not go blank when I click the "Stop Now" button.  Whatever current image, video, or effect current on the screen freezes and stays on the screen until I either enter and exit test mode, reset the panels, or start a sequence.

**Comment by ghormann** [comment]
If you set blank between preview on does the behavior change? 
![image](https://user-images.githubusercontent.com/24968726/89684005-9352f380-d8c7-11ea-851c-850ed224dedf.png)

**Comment by Cboehm00** [comment]
No, that didn't change anything.  I did that, and stopped it in the middle of a video and same result. ![image](https://user-images.githubusercontent.com/30914929/89686736-7c5ed200-d8c4-11ea-9a15-522d321c7e23.jpeg)

**Comment by ghormann** [comment]
To me, this part of Playlist.cpp looks odd.   If m_parent is true and exit=true (default), then SendBlackingData() will never be called.   
```
    if (m_parent && exit) {
        playlist = m_parent;
        if (m_parent->getPlaylistStatus() == FPP_STATUS_PLAYLIST_PAUSED) {
            m_parent->Resume();
        }
        PL_CLEANUPS.push_back(this);
    } else if (exit) {
        sequence->SendBlankingData();
    }
```

**Comment by dkulp** [comment]
If m_parent is non-null, that means it's returning to the parent playlist which will continue playing.   If we called blank at that point, we get a black flash as it goes from the child playlist back to the parent playlist.

If you just hit "play" from the UI, m_parent should be null.

**Comment by ghormann** [comment]
Thanks Dan for clarifying that.

@Cboehm00 : Two more questions: 

1. Is the FPP for your P5 in standalone or slave mode?
1. Are you starting a playlist or a single sequence from the UI?

**Comment by Cboehm00** [comment]
It's in standalone mode.  I'm starting a playlist from the UI.  

I've also tried a single sequence from the UI with the same result.  

If I output directly from xLights in bridge mode the P5 will go blank when stopping the sequence.

**Comment by ghormann** [comment]
@Cboehm00  I setup a BBB, and octoscroller, and a few P10 panels.   I'm unable to reproduce on the latest code in master.

Video attempting to reproduce: https://www.youtube.com/watch?v=chHErrwyPOc&feature=youtu.be

**Comment by Cboehm00** [comment]
This is a 49 panel setup with 2 colorlights and a Pi4.
I have a smaller panel with a BBB that I haven't tested yet.  I will test later today and report back.

Thanks!

**Comment by Cboehm00** [comment]
@ghormann 
The problem doesn't occur on a bbb or a colorlight on a smaller panel (9).  For some reason it's only happening on my large panel (49) with 2 colorlight cards.  I've swapped colorlights, Pis, reimaged SD, and I still get the same result on the large panel.

**Comment by ghormann** [comment]
Unfortunately, I don't have any colorlight cards try and reproduce.   The only thing I can personally think of is to put into debug level logging and then look at the logs around the stop to see if anything is difference between those that work and those that don't. 

Maybe others have additional ideas.

**Comment by dkulp** [comment]
This was likely fixed as part of the other colorlight fixes on master.

**Comment by ghormann** [comment]
@Cboehm00 I'm closing as Dan believes this was fixed in 4.2.   If not resolved, please comment.
